pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -67,6 +67,8 @@ rec {
     sha256_64bit = "09qcdfn4j5jza3iw59wqwgq4a489qf7kx355yssrcahaw9g87lxz";
     settingsSha256 = "16qqw0jy31da65cdi17y3j2kcdhw09vra7g17bkcimaqnf70j0ni";
     persistencedSha256 = "1ad81y4qfpxrx0vqsk81a3h0bi1yg8hw5gi5y5d58p76vc8083i9";
+
+    broken = with kernel; kernelAtLeast "5.17";
```

Comment:
```suggestion
    broken = kernel.kernelAtLeast "5.17";
```

Reply:
fixed this as well as the same line for `legacy340`, then squashed it to a single commit.